client/index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <title>Glint</title>
-  <link rel="stylesheet" href="styles/styles.css">
   <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" href="bower_components/fontawesome/css/font-awesome.min.css">
   <link rel="stylesheet" href="bower_components/animate.css/animate.min.css">
+  <link rel="stylesheet" href="styles/styles.css">
```

Comment:
I think this is a good example. You should always put your personal stylings last so that it overrides the default.